content/posts/install/index.md

```diff
@@ -0,0 +1,37 @@
++++
+"author" = "Rizin"
+"title" = "Install"
+"date" = "2022-08-17"
+"summary" = "How to install Rizin and Cutter"
+"tags" = ["rizin"]
+
+"ShowToc" = true
+"TocOpen" = false
+"weight" = 2
++++
+
+## Via package managers
+
+The easiest way to get Rizin and Cutter is to install from an operating system
+package manager.
+
```

Comment:
Probably worth mentioning OBS?
@ret2libc

Reply:
The allow edits by maintainers box is checked, so please push a commit :)